### PR TITLE
[compiler-rt] eventfd api interception on freebsd.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_freebsd.h [hunk 1]
```diff
@@ -726,6 +726,8 @@ struct __sanitizer_cpuset {
 
 typedef struct __sanitizer_cpuset __sanitizer_cpuset_t;
 extern unsigned struct_cpuset_sz;
+
+typedef unsigned long long __sanitizer_eventfd_t;
```
Comment (vitalybuka) [inline compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_freebsd.h:730]: if it's the same time as posix, can you have just a single typedef there in part enabled for freebsd as well?

**Review by vitalybuka — APPROVED** [review]

**Comment by devnexen** [comment]
in fact the posix one would need more changes, ie [freebsd does not really use it](https://github.com/llvm/llvm-project/blob/273aefec66cea947787f053adee25175f4168713/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h#L17) (e.g. __sanitizer_sigaction defined in the freebsd one but "defined" in the posix too).

**Comment by vitalybuka** [comment]
This bot is broken https://lab.llvm.org/buildbot/#/builders/77/builds/33380

**Comment by devnexen** [comment]
right. fixing soon, thx.

**Pull request merged**